### ISSUE TITLE
search option, case sensitivity issue

### ISSUE THREAD

**Issue by danrobi11** [DEVLOPRR/bread#2]
Searching for `vlc` return this:

```
$ /home/danrobi/.local/bin/bread-0.4.2-x86_64.AppImage search vlc
Getting Latest List...
Searching List... 100% |███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| (1227/1227, 391720 it/s)

```

It wont return any result, unless I search for `VLC`. Same for `Firefox`, no result if searching for `firefox` :/

**Comment by pegvin** [comment]
Thanks For Reporting This Issue, It Has Been Fixed Now In [v0.4.3](https://github.com/DEVLOPRR/bread/releases/tag/v0.4.3)

**Comment by danrobi11** [comment]
> It Has Been Fixed Now In [v0.4.3](https://github.com/DEVLOPRR/bread/releases/tag/v0.4.3)

Thank you much :+1:  Closing
